tests/visualization_tests/test_pareto_front.py

```diff
@@ -1,451 +1,230 @@
     )
-    actual_axis_order = axis_order or [0, 1, 2]
-    data: List[Tuple[int, ...]]
-    if use_constraints_func:
-        assert len(figure.data) == 3
-        if include_dominated_trials:
-            # The enqueue order of trial is: infeasible, feasible non-best, then feasible best.
-            data = [(1, 1, 1, 1), (1, 0, 1, 1), (1, 2, 2, 0)]
-        else:
-            # The enqueue order of trial is: infeasible, feasible.
-            data = [(1, 1, 1), (1, 0, 1), (1, 2, 0)]
-    else:
-        assert len(figure.data) == 2
-        if include_dominated_trials:
-            # The last elements come from dominated trial that is enqueued firstly.
-            data = [(1, 1, 1, 1), (0, 1, 1, 1), (2, 0, 2, 1)]
-        else:
-            data = [(1, 1), (0, 1), (2, 0)]
-
-    _check_data(figure, "x", data[actual_axis_order[0]])
-    _check_data(figure, "y", data[actual_axis_order[1]])
-    _check_data(figure, "z", data[actual_axis_order[2]])
-
-    titles = ["Objective {}".format(i) for i in range(3)]
-    assert figure.layout.scene.xaxis.title.text == titles[actual_axis_order[0]]
-    assert figure.layout.scene.yaxis.title.text == titles[actual_axis_order[1]]
-    assert figure.layout.scene.zaxis.title.text == titles[actual_axis_order[2]]
-
-    # Test with `target_names` argument.
-    error_message = "The length of `target_names` is supposed to be 3."
-
-    with pytest.raises(ValueError, match=error_message):
-        plot_pareto_front(
-            study=study,
-            target_names=[],
-            include_dominated_trials=include_dominated_trials,
-            axis_order=axis_order,
-            targets=targets,
-        )
-
-    with pytest.raises(ValueError, match=error_message):
-        plot_pareto_front(
-            study=study,
-            target_names=["Foo"],
-            include_dominated_trials=include_dominated_trials,
-            axis_order=axis_order,
-            targets=targets,
-        )
-
-    with pytest.raises(ValueError, match=error_message):
-        plot_pareto_front(
-            study=study,
-            target_names=["Foo", "Bar"],
-            include_dominated_trials=include_dominated_trials,
-            axis_order=axis_order,
-            targets=targets,
-        )
-
-    with pytest.raises(ValueError, match=error_message):
-        plot_pareto_front(
-            study=study,
-            target_names=["Foo", "Bar", "Baz", "Qux"],
-            include_dominated_trials=include_dominated_trials,
-            axis_order=axis_order,
-            targets=targets,
-        )
 
-    target_names = ["Foo", "Bar", "Baz"]
-    figure = plot_pareto_front(study=study, target_names=target_names, axis_order=axis_order)
-    assert figure.layout.scene.xaxis.title.text == target_names[actual_axis_order[0]]
-    assert figure.layout.scene.yaxis.title.text == target_names[actual_axis_order[1]]
-    assert figure.layout.scene.zaxis.title.text == target_names[actual_axis_order[2]]
```

Comment:
`_get_pareto_front_info` does not depend on the number of dimensions, so we do not need to check both 2D and 3D cases. We test both 2D and 3D cases of `_get_pareto_front_plot in test_get_pareto_front_plot`.